command/daemon.go

```diff
@@ -21,36 +21,47 @@ var DaemonCmd = &cli.Command{
 	Action: daemonCommand,
 }
 
-func daemonCommand(c *cli.Context) error {
-	ctx, cancel := context.WithCancel(ProcessContext())
-	defer cancel()
+func daemonCommand(cctx *cli.Context) error {
+	// TODO: get these from config file
```

Comment:
Does this mean that we'll remove CLI flags or that we are going to make them optional as a way to overwrite the current config?

Reply:
We will keep them to override configured values where it makes sense.